contracts/InterchainTokenFactory.sol

```diff
@@ -165,7 +167,8 @@ contract InterchainTokenFactory is IInterchainTokenFactory, ITokenManagerType, M
      * @notice Deploys a remote interchain token on a specified destination chain.
      * @param originalChainName The name of the chain where the token originally exists.
      * @param salt The unique salt for deploying the token.
-     * @param minter The address to distribute the token on the destination chain.
+     * @param minter The address to distribute the token on the destination chain. If the address is zero,
+     * no additional minter is set on the token. Reverts if the minter is not authorized or interchainTokenService address (invalid).
```

Comment:
same as above. `is not authorized` -> `does not have mint permission for the token`.

Reply:
got it, updated.